src/scanners/css.js

```diff
@@ -12,7 +12,7 @@ export default class CSSScanner extends BaseScanner {
 
   _defaultRules = rules;
 
-  processCode(cssCode, cssInstruction) {
+  processCode(cssCode, cssInstruction, rules) {
```

Comment:
Actually should this use _defaultRules like this: https://github.com/mozilla/addons-validator/blob/pass-metadata-to-rules-190/src/scanners/css.js#L51